src/NuGet.Server/Infrastructure/JsonNetPackagesSerializer.cs

```diff
@@ -21,15 +24,30 @@ public void Serialize(IEnumerable<ServerPackage> packages, Stream stream)
         {
             using (var writer = new JsonTextWriter(new StreamWriter(stream, Encoding.UTF8, 1024, true)))
             {
-                _serializer.Serialize(writer, packages.ToList());
+                _serializer.Serialize(
+                    writer,
+                    new SerializedServerPackages
+                    {
+                        SchemaVersion = CurrentSchemaVersion,
```

Comment:
>CurrentSchemaVersion [](start = 40, length = 20)

but what if the version is "1.0.0" (semver 1)? Will this mean we store it as semver 2, and old clients can't see it anymore?